pages/common/csvformat.md

```diff
@@ -0,0 +1,24 @@
+# csvformat
+
+> Convert a CSV file to a custom output format.
+> Included in csvkit.
+
+- Convert to a tab-delimited file:
+
+`csvformat -T {{data.csv}}`
+
+- Convert delimiters to a custom character:
+
+`csvformat -D "{{custom_character}}" {{data.csv}}`
+
+- Convert line endings to carriage return + line feed:
+
+`csvformat -M "{{\r\n}}" {{data.csv}}`
+
+- Convert to minimal use of quote characters:
+
+`csvformat -U 0 {{data.csv}}`
+
+- Convert to maximum use of quote characters:
```

Comment:
You don't have to wrap all fields in quote characters. If they have spaces, you need to use quotes or escape the spaces. It's the same as how any other *nix command handles spaces, so I didn't think it needed an extra explanation.